charts/ark-solr/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: solr-helm-charts
```

Comment:
@naveen-armedia I believe the name of the Helm chart must be exactly the same as the directory it is located in, `ark-solr` here. See [here](https://github.com/helm/helm/issues/1979).

Reply:
I am not aware of it. Thanks, updated with same name as the directory.